src/DbAuth.cc

```diff
@@ -43,9 +43,9 @@ static char const* SVNid __attribute__((unused)) = "$Id$";
 #include "lsst/daf/base/Citizen.h"
 #include "lsst/daf/persistence/DbAuth.h"
 
-#include "boost/scoped_array.hpp"
 #include <cstdlib>
 #include <fstream>
+#include <memory>
```

Comment:
Maybe worth updating the copyright date in this file (and elsewhere) when you touch it.